typings/@emotion/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// Stub @emotion/core
+// Remove when @storybook has moved to @emotion v11
```

Comment:
Should we link the comment to the issue in the Storybook repo?

Reply:
Done